roles/oVirt.datacenter-cleanup/tasks/find_hosted_engine_details.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Find existing VMs
+  ovirt_vms_facts:
+    auth: "{{ ovirt_auth }}"
+    pattern: "datacenter={{ data_center_name }}"
+
+- name: Find HE info if HE
+  set_fact:
+    hosted_engine: "{{ ovirt_vms | selectattr(search_key, 'equalto', search_val) | list | count > 0 }}"
+  vars:
+    search_key: origin
+    search_val: managed_hosted_engine
```

Comment:
Should we search here also for `hosted_engine`?

Reply:
we set hosted_engine to True if this condition is true. 
I maybe didn't understand your question.